newsletter/2020/09-18.html

```diff
@@ -0,0 +1,266 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Newsletter September 18, 2020</title>
+  <link rel="stylesheet" href="../../style.css">
+  <link rel="stylesheet" href="../../newsletter-form.css">
+</head>
+
+<body>
+  <div class="hero">
+    <h1 class="header hero-header">
+      <a class="hero-header-link" href="/">
+        Operation Spark <br /> Alumni Newsletter
+      </a>
+    </h1>
+    <div>
+      <img class="hero-logo" src="../../assets/logo-white.svg" alt="Hallebot Head with Graduation Cap">
+    </div>
+  </div>
+  <div class="section">
+    <!-- <h1 class="h1">Operation Spark Alumni Newsletter</h1> -->
+    <p>September 18, 2020</p>
+
+    <h2>
+    Letter from a code editor
+    </h2>
+    <p>
+      Dear readers,
+    </p>
+    <p>
+      Thank you for your interest in the Operation Spark Alumni Newsletter! In this newsletter we aim to share resources
+      that we think are useful, interesting, or just awesome. This month’s content comes from the alumni board, but if
+      you read to the end we have a form for alum suggestions! We want to hear what you think is cool and exciting so we
+      can share it with the rest of our community.
+    </p>
+
+    <hr class="separator" />
+    <h2>Learning/Practice Resource</h2>
+    <h3>
+      Learning by implementing
+    </h3>
+    <p>
+      There can, at times, be a sense that the greatest software engineers create their own stuff and other people join
+      in and work on it. However, one of the best ways to grow as an engineer is by reimplementing things that have
+      already been done, and then comparing your solution to theirs. Here is a quote from The Coding Career Handbook
+      that explains this really well:
+    </p>
+    <p>
+      “Your decision-making is a scarce resource. You start every day with a full tank, and as you make decisions
```

Comment:
Not sure how this is formatted in the source, but I think it would be more readable if we broke up the quote into more paragraphs

Reply:
Good idea, split into three paragraphs